src/Resources/app/storefront/src/checkout/confirm-order.plugin.js

```diff
@@ -0,0 +1,64 @@
+import Plugin from 'src/plugin-system/plugin.class';
+import DomAccess from 'src/helper/dom-access.helper';
+import StoreApiClient from 'src/service/store-api-client.service';
+import FormSerializeUtil from 'src/utility/form/form-serialize.util';
+import ElementLoadingIndicatorUtil from 'src/utility/loading-indicator/element-loading-indicator.util';
+
+/* global adyenCheckoutOptions */
+/* eslint-disable no-unused-vars */
+export default class ConfirmOrderPlugin extends Plugin {
+
+    init() {
+        const confirmOrderForm = DomAccess.querySelector(document, '#confirmOrderForm');
+        confirmOrderForm.addEventListener('submit', this.confirmOrder.bind(this));
+    }
+
+    confirmOrder(event) {
+        event.preventDefault();
+        const form = event.target;
+        if (!form.checkValidity()) {
+            return;
+        }
+
+        this._client = new StoreApiClient();
+        const formData = FormSerializeUtil.serialize(form);
+
+        ElementLoadingIndicatorUtil.create(document.body);
+
+        const orderId = this.options.orderId;
+        const request = new XMLHttpRequest();
+        let callback = null;
+        if (typeof(orderId) !== 'undefined' && orderId !== null) { //Only used if the order is being edited
```

Comment:
I believe the same can be achieved with this but please double check with @KadoBOT 
```suggestion
        if (!!orderId) { //Only used if the order is being edited
```

Reply:
Thanks! 👍